fmcg_store/data/pos_category.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="pos_category_5" model="pos.category">
+        <field name="name">Food Grains</field>
+    </record>
```

Comment:
What is parent category for this?

Reply:
because they are the main categories, this all categories does not act as subcategories so they don't have any parent_id
Same goes with the below points also.